### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by sparsh9**
Hey, Please review the changes in made in your readme file.

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sansyrox) [inline .idea/workspace.xml:1]: What is this file?

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sansyrox) [inline .idea/vcs.xml:1]: What is this file?

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sansyrox) [inline .idea/modules.xml:1]: Same question

File: .idea/algorithm-ds.iml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sansyrox) [inline .idea/algorithm-ds.iml:1]: What is this file used for?

Comment (sparsh9) [reply .idea/algorithm-ds.iml:1]: I'm Sorry. These are just default files proviede by WebStorm, Please ignore them.

**Review by sparsh9 — COMMENTED** [review]
These are just files provided by WebStorm. You can simply ignore them.

**Comment by sansyrox** [comment]
@sparsh9 , please delete these files and update the PR 😄

**Comment by sparsh9** [comment]
> @sparsh9 , please delete these files and update the PR

yeah sure.

**Pull request closed without merging**